### ISSUE TITLE
Engage with Authenticate Experience on Authenticated Pages

### ISSUE THREAD

**Issue by jason-gcio** [department-of-veterans-affairs/va.gov-team#2380]
## Goal
Learn what the process or guidance is from the Auth Exp perspective to create an authenticated page for the Rated Disabilities Tool so that we can coordinate those pages.

## Considerations
- A [link to mockups](https://xd.adobe.com/view/0db723ac-52ff-48b3-4877-9d5882cb2e1f-7b86/screen/8990839d-80b5-450d-8164-c1f203a8991d/RD-Screen-Tool-2/) currently held
- Getting to these pages is unknown at this time but thinking they might come out of the disability hub on the profile page?

## Task
- Schedule a meeting with members of the Authenticated Experience Team (DSVA - Lisa and PM Justin)

## Acceptance Criteria
- [x] Schedule a meeting with Justin, Lisa and Samara

**Comment by Samara-Strauss** [comment]
@jason-gcio wanted to update you on how things are going here for the rated disabilities work:

- I've been in regular contact with @sporeboy about his design and approach. This seems to be working well and I plan to stay in the loop on the rated disabilities designs as they progress.
- I had originally been thinking I would link to the rated disabilities page from the profile, but I have changed that plan and we will no longer be taking that approach. Eventually, I do plan to link to the rated disability page from the new logged in homepage/dashboard that we'll be working on in a few months as part of personalization 2.0, but in the meantime, your team should work to figure out wayfinding and how to guide users to the rated disability page in both a logged in and logged out state. I am happy to review and discuss approaches as your team has them. I've talked to @sporeboy about this already and he's on board.

And re: general collaboration, I think it should work roughly as follows:

- When you're working on a new feature, make sure to reach out to our team so we're aware of the feature's development. I would imagine this might look like an initial convo or two so we can discuss high-level questions and potential approaches.
- After those initial convos, your team will conduct whatever research/design they see fit. I'd like to be kept in the loop on this from a knowledge/consulting perspective, but I don't plan on telling anyone how to do their work.
- When the designer on the team — @sporeboy and whomever else — has designs to share, we can discuss those as they iterate. We can also continue to discuss things like how we'll link and guide users to features.

How does that sound. Any questions?
